crates/store/re_data_source/src/data_source.rs

```diff
@@ -35,27 +34,26 @@ pub enum DataSource {
     #[cfg(not(target_arch = "wasm32"))]
     Stdin,
 
-    /// A `rerun://` URI pointing to a recording or catalog.
-    RerunGrpcStream {
-        uri: RedapUri,
+    /// A `rerun://` URI pointing to a recording.
+    RedapDataset {
```

Comment:
the problem with `RerunGrpcStream` was that this also contained a lot of URLs that are NOT log data sources. We're being a lot more explicit here now